README.md

```diff
@@ -39,6 +39,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [ArnabXD/AnimeDB-tgbot](https://github.com/ArnabXD/AnimeDB-tgbot) - Search and retrieve anime and manga.
 - [gab-palmeri/TagBot](https://github.com/gab-palmeri/TagBot) - Create tags to mention multiple group members at once. [ᴜsᴇ](https://t.me/grouptags_bot)
 - [vrumger/WhisperedBot](https://github.com/vrumger/WhisperedBot) - Send whispers to people in groups so no one else can read them. [ᴜsᴇ](https://t.me/WhisperedBot)
+- [voxelin/unihook](https://github.com/voxelin/unihook) - Handle automatic webhooks right in telegram. [ᴜsᴇ](https://t.me/uniwebhookbot)
```

Comment:
Can you provide some more context?

Reply:
Detailed info in bot's repo. This bot creates a webhook listener path on `hook.ieljit.lol` domain and if gets request, parses it by user-given template and send message to a chat.

Reply:
Can you, please, update the description to be a little more descriptive so the reader does not have to manually open the project's README for more specifics?